Library/Homebrew/build.rb

```diff
@@ -81,6 +82,7 @@ def install
     if superenv?
       ENV.keg_only_deps = keg_only_deps
       ENV.deps = formula_deps
+      ENV.run_time_deps = run_time_deps
       ENV.x11 = reqs.any? { |rq| rq.is_a?(X11Requirement) }
```

Comment:
Tangent: Does this need to be changed to also support XorgRequirement?

Note that when we port that to Homebrew/brew, I think we'll probably drop that name and instead use the extension mechanism Mike made, which is going to require some rather substantive retesting on our end come merge time.